javascript-bank/account.js

```diff
@@ -0,0 +1,42 @@
+/* exported Account */
+
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+Account.prototype.deposit = function (amount) {
+  var transaction = new Transaction('deposit', amount);
+  if (amount > 0) {
+    this.transactions.push(transaction);
+    return true;
+  } else {
+    return false;
+  }
+};
+
+Account.prototype.withdraw = function (amount) {
+  var transaction = new Transaction('withdrawl', amount);
+  if (amount > 0) {
+    this.transactions.push(transaction);
+    return true;
+  } else {
+    return false;
+  }
+};
+
+Account.prototype.getBalance = function () {
+  var funds = 0;
+  if (this.transactions.length === 0) {
```

Comment:
This conditional is unnecessary. If the length of the array is zero, your loop following this will end immediately, and leave you in the same situation as your condition is preparing for :+1: